README.md

```diff
@@ -32,11 +32,11 @@
 <!-- mega-linter-badges-end -->
 
 <!-- welcome-phrase-start -->
-MegaLinter is an **Open-Source** tool for **CI/CD workflows** that analyzes the **consistency of your code**, **IAC**, **configuration**, and **scripts** in your repository sources, to **ensure all your projects sources are clean and formatted** whatever IDE/toolbox is used by their developers, powered by [**OX Security**](https://www.ox.security/?ref=megalinter).
+MegaLinter is an **open-source** tool for **CI/CD workflows** that analyzes the **consistency of your code**, **IaC**, **configuration**, and **scripts** in your repository to **ensure all your project sources are clean and formatted**, regardless of the IDE or tools used by your developers. Powered by [**OX Security**](https://www.ox.security/?ref=megalinter).
 
-Supporting [**65** languages](#languages), [**22** formats](#formats), [**20** tooling formats](#tooling-formats) and **ready to use out of the box**, as a GitHub action or any CI system, **highly configurable** and **free for all uses**.
+Supporting [**65** languages](#languages), [**22** formats](#formats), and [**20** tooling formats](#tooling-formats). It is **ready to use out of the box** as a GitHub Action or with any CI system, **highly configurable**, and **free for all uses**.
```

Comment:
Are these numbers still updated for the next release?

Reply:
... and i've not updated the grammar in the build script :'(